### ISSUE TITLE
Feature Request - Toggle Specific Mods Checking for Updates ON/OFF

### ISSUE THREAD

**Issue by ChemGuy1611** [Nexus-Mods/Vortex#16346]
**Is your feature request related to a problem? Please describe.**
I have a mod that has been updated, but the update has a bug that makes the mod unusable. Every time I check for updates in Vortex, I have to select "Ignore Update" in the context menu for that mod (This function was added using "Clear Update" extension) to remove it from the "Update available" list. 

**Describe the solution you'd like**
I would like an option added to the context menu to toggle update checks ON and OFF for specific mods. That way, I avoid seeing updates for that mod.

**Describe alternatives you've considered**
The alternative now is to manually ignore the update every time. 

**Additional context**
N/A

**Comment by ak1dd** [comment]
I see another use case here that is all too common.  A mod is removed from Nexus (or whatever) and you want to keep it anyway.  It is never going to be updated again because if it is re-posted it won't be identified as the same mod anyway.  This happens often in bigger modlists.

**Comment by ChemGuy1611** [comment]
Another real use case for this would be on Fallout 4, if you play "Old-Gen" version (as many enthusiast modders do), You have to be very cautious to avoid accidentally updating mods to their Next-Gen (NG) versions, as many mod authors use newer version numbers on those files.

Updating mods in batch becomes impossible in this scenario, because you have to interrogate each mod to make sure it is not trying to update to a NG version

**Comment by Dorennor** [comment]
> Hey @ChemGuy1611,
> 
> I'm personally not a fan of obfuscating functionality (especially when it comes to keeping my mods up to date) but that's just personal preference. From a coding perspective, this will not be a trivial task due to how the update chain is propagated on the site and the fact that sometimes Vortex just doesn't know which file to pick (reason why it will sometimes require you to go to the nexus mods page and select the wanted file). Ignoring the update under those circumstances has the potential to never notify you of future updates for that mod again until it's re-installed; which may raise questions from the users "why is my mod not updating?" (it's easy to forget about a mod you've ignored an update for months ago).
> 
> Why not use Vortex's highlight tool to clearly mark the mod?
> <img width="1123" alt="image" src="https://github.com/user-attachments/assets/ddb4b4f2-baf5-4f9a-a4d9-372edcfe091c">
> 
> Given that I find both our views subjective, and the fact that this is a piece of functionality embedded in Vortex's nexus integration module. Going to ask @insomnious whether to allocate time for this in the future.

#23219 here.
So, here is one of my use cases: I have a mod for RDR2 which has a broken link/versioning or something. Basically, it always says there's an update, but it's a false positive - it just detects an additional file as an update for the main file or something. Because of that, the auto-update feature is broken, and I can, by mistake, update this mod just by pressing some buttons.

A permanent "ignore update" feature or some kind of tag, mark, or anything to separate it from other updates is welcome. Because currently, broken updates can cause issues for some of my games (RDR2, Valheim, Lethal Company, Cyberpunk, Fallout 4, etc.).

**Comment by ChemGuy1611** [comment]
Yes, that is exactly the kind of use case I was thinking of. That, and avoiding installing Next-Gen updates for FO4 mods 😂
